src/database/queries/canonicalFields/fetchAll.sql

```diff
@@ -0,0 +1,7 @@
+SELECT
+  canonical_fields.id as "id",
```

Comment:
Is it needed to qualify `canonical_fields.` here or following?

Reply:
Probably not; I've gotten in the habit of it just from a "explicit is nice and consistency regardless of when you JOIN is nice too" perspective but I don't feel strongly and totally fine being more terse.

Reply:
I see that perspective too and it appeals to me. Not a big deal either way. But I am thinking if I write some SQL, though, I will find myself doing the `from canonical_fields cf`, so I suppose that's my preference, maybe more out of habit and/or laziness and/or trying to keep lines under 80 chars (though here there is no problem).

Reply:
`from canonical_fields cf` it shall be :D